sdv/lite/tabular.py

```diff
@@ -86,13 +91,14 @@ def fit(self, data):
             data (pandas.DataFrame):
                 Data to fit the model to.
         """
-        self._null_percentages = {}
-
-        for column, column_data in data.iteritems():
-            num_nulls = column_data.isna().sum()
-            if num_nulls > 0:
-                # Store null percentage for future reference.
-                self._null_percentages[column] = num_nulls / len(column_data)
+        if not self._null_column:
```

Comment:
can we add the test case for when this should not learn the null percentages?

Reply:
@amontanez24 That should be covered in the `test_fit` unit test, and I also assert that null percentages is null at the end.

Reply:
Added!